### DISCUSSION TITLE
Can you have more than one discussion type in a repo?

### DISCUSSION THREAD

**Discussion by rfalanga** [community/community#128641] [Discussions] [answered]
### Select Topic Area

Question

### Body

I'm going through the Introduction to GitHub skills course. I've known about Discussions for a while but going through that course I now realize there's more to discussions than I thought. I had thought that discussions were only for the members of the repo. I now know that discussions are a way for other people, outside of the members of the repo, to ask questions and interact with members of the repo. I'm wondering, if it possible to have more than one discussion type in a repo's Discussion area?

**Comment by davevad93** [comment, marked as answer]
Hi @rfalanga ,

Yes it's possible to have more than one discussion type in a repo, you can choose between:

1. 📣Announcements
2. 💬General
3. 💡Ideas
4. 🗳️Polls
5. 🙏Q&A
6. 🙌Show and tell

You can also manage the discussion categories by adding new sections or categories by clicking on `edit`.

![0](https://github.com/community/community/assets/94308234/ce2dd7b4-8337-466e-9600-c4b583a795a9)
